services/web/server/src/simcore_service_webserver/projects/projects_api.py

```diff
@@ -107,8 +107,11 @@ def validate_project(app: web.Application, project: Dict):
 
 
 async def delete_project(request: web.Request, project_uuid: str, user_id: str) -> None:
-    await remove_project_interactive_services(user_id, project_uuid, request.app)
-    await delete_project_data(request, project_uuid, user_id)
+    async def remove_services_and_data():
+        await remove_project_interactive_services(user_id, project_uuid, request.app)
+        delete_project_data(request, project_uuid, user_id)
```

Comment:
```suggestion
        await delete_project_data(request, project_uuid, user_id)
```
if you do not await, then the coroutine is not executed